tests/integration/cairo1_compiler_bindings/print_test.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import Path
+
+import pytest
+
+from tests.integration._conftest import ProtostarProjectFixture
+from tests.integration.conftest import (
+    CreateProtostarProjectFixture,
+    assert_cairo_test_cases,
+)
+from protostar.testing.test_results import PassedTestCaseResult
+
+
+@pytest.fixture(name="protostar_project", scope="function")
+def protostar_fixture(create_protostar_project: CreateProtostarProjectFixture):
+    with create_protostar_project() as protostar_project:
+        yield protostar_project
+
+
+async def test_print(protostar_project: ProtostarProjectFixture, datadir: Path):
+    protostar_project.create_contracts_cairo1(
+        {
+            "simple": datadir / "simple",
+        }
+    )
+    testing_summary = await protostar_project.protostar.run_test_runner(
+        datadir / "basic_print_test.cairo",
+        cairo1_test_runner=True,
+    )
+
+    assert_cairo_test_cases(
+        testing_summary,
+        expected_passed_test_cases_names=["test_print_basic", "test_print_in_contract"],
+    )
+
+    expected_outputs = {
+        "test_print_in_contract": [
+            "original value: [123]",
+            "original value: [3]",
+            "original value: [24]",
+            "original value: [72]",
+        ],
+        "test_print_basic": [
+            "original value: [448378203247] converted to a string: [hello]",
+            "original value: [439721161573] converted to a string: [false]",
+            "original value: [1953658213] converted to a string: [true]",
```

Comment:
Don't we want to assert that other prints (1.print(), array.print()) also work?

Reply:
sure why not

Reply:
[naming](https://github.com/software-mansion/protostar/pull/1803/commits/8c651fda6d4627194766a8aa9a452fc3506e169f)